src/components/auth/login.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Login from './login';
+import { clikLogin } from "../../utils/test-utils";
+
+
+describe('Button', () => {
+  it('should be defined', () => {
+    expect(Login).toBeDefined();
+  });
+
+  it('should render correctly', () => {
+    const tree = (
+      <Login name='button test' />
+    );
+    expect(tree).toMatchSnapshot();
+  });
+ });
+
+ test('adds properly', () => {
```

Comment:
are we creating a sum of something (numerical values) in login. I'm not quite sure what this is testing for that affects the login page.